docs/auth/social-auth.md

```diff
@@ -400,26 +407,29 @@ with the new authentication state of the user.
 
 > If you enabled the **One account per email address** setting in the Firebase console, when a user tries to sign in a to a provider (such as Google) with an email that already exists for another Firebase user's provider (such as Facebook), the error `auth/account-exists-with-different-credential` is thrown along with an `AuthCredential` object (Google ID token). To complete the sign in to the intended provider, the user has to sign first to the existing provider (Facebook) and then link to the former `AuthCredential` (Google ID token).
 
-To provide users with an additional login method, you can link their social media account (or an email & password) with their Firebase account. This is possible for any social provider that uses `auth().signInWithCredential()`.
-To achieve this, you should replace sign-in method in any of the supported social sign-in code snippets with `auth().currentUser.linkWithCredential()`.
+To provide users with an additional login method, you can link their social media account (or an email & password) with their Firebase account.
+This is possible for any social provider that uses `signInWithCredential()`.
+To achieve this, you should replace sign-in method in any of the supported social sign-in code snippets with `getAuth().currentUser.linkWithCredential()`.
```

Comment:
currentUser issue as mentioned above - should be documented more like "replace with `linkWithCredential(credential.user, credential)`"

(combining the idea that currentUser doesn't exist in modular, so it should be referenced from the previous credential you signed in with, used in combination with the new credential you want to link to the user)

https://firebase.google.com/docs/reference/js/auth#linkwithcredential_60f8043